src/Tizen.MachineLearning.Nnstreamer/Tizen.MachineLearning.Nnstreamer/TensorsData.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace Tizen.MachineLearning.Nnstreamer
+{
+    /// <summary>
+    /// The TensorsData class sets and gets the buffer data for each Tensor.
+    /// </summary>
+    /// <since_tizen> 5.5 </since_tizen>
+    public class TensorsData : IDisposable
+    {
+        private IntPtr _handle = IntPtr.Zero;
+        private bool disposed = false;
+
+        /// <summary>
+        /// Creates a TensorsInfo instance with handle which is given by TensorsInfo.
+        /// </summary>
+        /// <param name="handle">The handle of tensors data.</param>
+        /// <since_tizen> 5.5 </since_tizen>
+        public TensorsData(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Destructor of the TensorsData instance
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
+        ~TensorsData()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Sets a tensor data to given index.
+        /// </summary>
+        /// <param name="index">The index of the tensor.</param>
+        /// <param name="buffer">Raw tensor data to be set.</param>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 5.5 </since_tizen>
+        public void SetBuffer(int index, byte[] buffer)
```

Comment:
How about rename to `SetTensorData`, description also `Sets a tensor data`

Reply:
I will.